src/components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ListGroup, Card } from 'react-bootstrap';
+
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+type Props = {
+  comments: PostComment[],
+};
+
+export const CommentList: React.FC<Props> = ({
+  comments,
+}) => (
+  <ListGroup as="ul">
+    {comments?.map((postComment: PostComment) => (
```

Comment:
comments не может быть null
```suggestion
    {comments.map((postComment) => (
```